app/src/pages/index.tsx

```diff
@@ -21,7 +28,7 @@ const Index: NextPage = () => {
         ))}
       </ul>
       <Trans i18nKey="Index.time" />
-      <ButtonLink href="/how-it-works" labelKey="Index.button" />
+      <ButtonLink href={forwardRoute} labelKey="Index.button" />
     </>
   )
 }
```

Comment:
Notes for reviewers: The index page (`/`) is the homepage for the eligibility screener. It is a content-only page that has no back link. The call to action button routes the user to `/how-it-works`. There are no data validation guards for this page because it is the first page in the form wizard flow.

Note that if you check out _this commit_ and attempt to run the application, the application would fail because it relies on updated props (i.e. `forwardRoute`) passed in from `_app.tsx` that aren't introduced until a later commit.

![Screen Shot 2022-10-28 at 1 31 12 AM](https://user-images.githubusercontent.com/67701/198542668-d1d353d1-e945-432a-8ac9-98b53fe628eb.png)